manpages.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+var manpagesMatcher = regexp.MustCompile(`\b(\w+)\((\d+)\)(\W|$)`)
```

Comment:
Strictly speaking, there are sections which aren’t strictly numerical, e.g. https://manpages.debian.org/jessie/libedit-dev/editline.3edit. A more practical example is 3pl. That said, I expect that in practice, references will usually be restricted to numerical sections. So, maybe add a TODO?